src/features/flows/nRF9151/Evaluate.tsx

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
+ */
+
+import React, { useState } from 'react';
+import {
+    Button,
+    logger,
+    openUrl,
+    Spinner,
+    telemetry,
+} from '@nordicsemiconductor/pc-nrfconnect-shared';
+import describeError from '@nordicsemiconductor/pc-nrfconnect-shared/src/logging/describeError';
+
+import { useAppDispatch, useAppSelector } from '../../../app/store';
+import { Back } from '../../../common/Back';
+import Copy from '../../../common/Copy';
+import { formatResponse } from '../../../common/formatATResponse';
+import Main from '../../../common/Main';
+import { Next } from '../../../common/Next';
+import { IssueBox } from '../../../common/NoticeBox';
+import { getSelectedDeviceUnsafely } from '../../device/deviceSlice';
+import getUARTSerialPort from '../../device/getUARTSerialPort';
+import {
+    getAttestationToken,
+    getFailed,
+    setAttestationToken,
+    setFailed,
+} from './nrf9151Slice';
+
+const nRFCloudLink =
+    'https://docs.nordicsemi.com/bundle/nrf-cloud/page/GettingStarted.html';
+
+export default () => {
+    const dispatch = useAppDispatch();
+    const device = useAppSelector(getSelectedDeviceUnsafely);
+    const token = useAppSelector(getAttestationToken);
+    const failed = useAppSelector(getFailed);
+    const [gettingToken, setGettingToken] = useState(false);
+
+    return (
+        <Main>
+            <Main.Content
+                heading="Evaluate nRF Cloud multi-service sample"
+                subHeading="Onboarding, data collection, FOTA updates, location, logging, alerts, and more."
+            >
+                <div className="tw-flex tw-flex-col tw-gap-4">
+                    <div>
+                        <b>Claim your device</b>
+                        <p>
+                            To evaluate all features you need to connect your DK
+                            to nRF Cloud. Copy the attestation token below and
+                            follow the instructions to claim your DK with nRF
+                            Cloud.
```

Comment:
```suggestion
                            To evaluate all features, you need to connect your DK
                            to nRF Cloud. Copy the attestation token below and
                            follow the instructions.
```
What does it mean to `claim your DK with nRF Cloud`? In the link provided, I can only dig out "add" or "associate".

Reply:
Not sure. Both my contacts for this are ooo so I'll have to check this up.

Reply:
Sure. Also, while I don't like `claim`, which is mostly used in legal contexts, I am pretty sure you cannot claim something `with` a software platform, but rather with an organization. So `claim in nRF Cloud` rather than `with`, unless you need to interact with a representative of nRF Cloud. But I would drop `claim with` altogether, as it reads weird.

Reply:
I'd say we just go with your initial suggestion for now.
I don't expect to get feedback on this soon. Worst case this is something simple to fix before release.
Or would you prefer to wait with this?

Reply:
Just go with my suggestion.